services/brig/src/Brig/Options.hs

```diff
@@ -21,7 +24,7 @@ import qualified Data.Yaml   as Y
 
 newtype Timeout = Timeout
     { timeoutDiff :: DiffTime
-    } deriving (Eq, Enum, Ord, Num, Real, Fractional, RealFrac, Show)
+    } deriving newtype (Eq, Enum, Ord, Num, Real, Fractional, RealFrac, Show)
```

Comment:
Look up `DerivingStrategies`; it's a bit tough to find real docs on; but it's a simple enough concept. Just allows you to tell it how you want it to do the derivation; here I'm saying to explicitly use `GND` rather than some other strategy (e.g. DeriveAnyClass)